README.md

```diff
@@ -0,0 +1,72 @@
+# Conduit Connector Google Cloud Platform Pub/Sub
+
+### General
+The GCP Pub/Sub connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides both, a source and destination GCP Pub/Sub connector.
```

Comment:
There's the default Pub/Sub service and also the Pub/Sub Lite service. IIUC, the APIs are the same and this connector supports both?

Reply:
No, APIs are not the same.
So this connector supports GCP Pub/Sub only (not the `lite` version).

Reply:
It would be good to note that.

Reply:
Actually, it was my bad that I didn't check it in the investigation phase, so I'll send you more information about the `lite` version.

Reply:
Haris, I've added a Pub/Sub Lite service to a current connector.
https://github.com/conduitio-labs/conduit-connector-gcp-pubsub/pull/29